README.md

```diff
@@ -1,43 +1,12 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-
 <a name="readme-top"></a>
+<div align="center">
+  
+  <img src="./Logo HG.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-<!--
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages capitalized to make your pull request professional and descriptive 👍